src/d3-ext/profile.js

```diff
@@ -58,13 +62,13 @@ ngeo.profile = function() {
    * @type {function(Object)}
    * Hover callback function.
    */
-  var hoverCallback = goog.nullFunction;
+  var hoverCallback = options.hoverCallback || goog.nullFunction;
```

Comment:
> Honestly, having options.hoverCallback repeated makes it harder to read and easy to make typos.

Come on! You really can't say that `var foo = goog.isDef(options.foo) ? options.foo : "default"` is hard to read.

> I disagree with you on the second point: I expect all developers working on Ngeo to understand basics concepts of javascript.

People very often make mistakes because `undefined`, `null`, `0` and `''` are all falsy values. People who see `var callback = options.callback || goog.nullFunction` will do the same for their options of type `number`: `var myNum = options.myNum || -1`.

I think it's a good thing to teach people to be stricter and more explicit with types.